packages/compute-baseline/src/baseline/index.ts

```diff
@@ -44,6 +44,33 @@ interface SupportStatus {
   toJSON: () => string;
 }
 
+/**
+ * Calculate a Baseline status for specific browser compat data keys within a
+ * web-features feature, in the style of a web-feature's `status` key. Use this
+ * method to calculate fine-grained support statuses. This is the only method
+ * approved to compute Baseline statuses not otherwise published in the
+ * `web-features` package.
+ *
+ * For example, suppose you want to show a Baseline status for a specific method
+ * in a feature, which might've been supported earlier or later than the broader
+ * feature overall. Then you'd call `getStatus('example-feature',
+ * 'api.ExampleManager.doExample')`.
+ */
+export function getStatus(
+  featureId: string,
+  compatKey: string,
+  compat: Compat = defaultCompat,
+) {
```

Comment:
Can you add the return type here? It looks like it's not the same as `computeBaseline`, and I wonder if that's intentional.

Reply:
Done with 63e00adaab8af7e5b4f6cdef5847207e60c8e02b. (Sorting out the peer dependency between `compute-baseline` and `web-features` will be fussy. Saving it for a PR _just_ for that.)